polaris-react/src/components/Grid/README.md

```diff
@@ -0,0 +1,147 @@
+---
+name: Grid
+category: Structure
+keywords:
+  - one column
+  - two column
+  - three column
+  - column
+  - row
+  - column layouts
+  - grid layouts
+  - containers
+  - full width containers
+  - css grid
+notice:
+  status: alpha
+  message: This component is in development. There could be breaking changes made to it in a non-major release of Polaris. Please use with caution.
+---
+
+# Grid
+
+Create complex layouts based on [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/grid)
+
+---
+
+## Examples
+
+### Two column wrapping layout
+
+Use to create a two column layout that wraps at a breakpoint and aligns to a twelve column grid.
+
+```jsx
+<Page fullWidth>
+  <Grid>
+    <Grid.Cell columnSpan={{xs: 2, sm: 2, md: 3, lg: 6, xl: 6}}>
```

Comment:
Should the numbers for `xs` and `sm` get changed in this example (in this case to 3) and in other examples below since we've switched to a 6 column grid for `xs` and `sm`?

Reply:
These are just column span examples, they can be whatever we want them to be. The underlying grid is set to match what we have in figma